documentation/docs/workflow.md

```diff
@@ -0,0 +1,142 @@
+
+### Branches
+IOTA Identity always has two permanent branches: `main` and `dev`. Both branches are protected and disallow direct commits; the only changes allowed are from pull requests approved and merged by maintainers.
+
+#### Main (main)
+The `main` branch contains a stable version of the code that is released towards package managers such as `crates.io` and `npm`. This branch only accepts PRs that merge from `release` or `hotfix` branches. 
+
+#### Dev (dev)
+The `dev` branch contains a frequently updated version of the code that is released towards package managers under a development flag. These releases may contain breaking changes without a strong notice towards developers using them. While the `dev` branch may get frequent updates, it may not contain unfinished features. Any large, multi-PR feature should be committed to a long-lived `epic` branch created specifically for that feature.
+
+### Work Branches
+
+These are branches that developers work on directly. Their names should be prefixed appropriately with one of the following categories. For example, a PR fixing a null pointer bug in the Wasm bindings might be created from a branch called `fix/client-non-null`.
+
+#### Feature (feat/, doc/, chore/, fix/)
+Singular PR contributions should create either a `feat`, `doc`, `chore`, or `fix` branch, depending on the type of changes. These may be branched from either the `dev` branch or an `epic` branch. If the number of lines of code is going to be relatively small and the work completed in a single PR, the branch should be created from `dev` and merged back into `dev` once completed. Otherwise, the branches should be created from their associated `epic` branch and be merged back into the same `epic` branch. 
+
+- `feat` branches should contain changes to the code that expand or modify functionality. They should also include updates to the documentation and examples related to the feature, though `doc` branches may be used to catch up on documenting a feature.
+- `doc` branches contain changes to code documentation or the wiki. These PRs should be kept relatively small in order to not burden a reviewer with too many documentation updates. For example, during a documentation catch-up, we will have a branch/PR per documentation page.
+- `chore` branches are short lived branches that contain no significant features or functionality changes, but rather smaller fixes such as typos, code fixes, minor refactors, and CI changes.
+- `fix` branches correct bugs such as compilation errors or where existing features do not behave as expected, generally without introducing any new functionality or breaking changes.
+
+It is recommended to integrate `dev` or `epic` regularly, depending on from where the branch started, to reduce the possibility and potential size of merge conflicts.
```

Comment:
```suggestion
It is recommended to integrate `dev` or `epic` regularly, depending on from where the branch started, to reduce the possibility and size of merge conflicts.
```

Reply:
@cycraig If changes are merged into the epic branch (by other developers) or `dev` is merged into the epic branch IMHO it makes sense to integrate them.